azurerm/resource_arm_storage_container.go

```diff
@@ -124,6 +125,18 @@ func resourceArmStorageContainerCreate(d *schema.ResourceData, meta interface{})
 	return resourceArmStorageContainerRead(d, meta)
 }
 
+func checkContainerIsCreated(reference *storage.Container) func() *resource.RetryError {
+	return func() *resource.RetryError {
+		createOptions := &storage.CreateContainerOptions{}
```

Comment:
Should we explicitly set timeout here so we can effectively use the timeout duration of the `Retry` call? I'm not sure on the default, maybe its ok?

Reply:
There’s sin inyl one other instance of retrying in the codebase that’s I found and the timeout was not configurable. We can add a create timeout to the scheme if we think it should be adjustable?

Reply:
we lean on the SDK's default timeout through the codebase, so I don't think we need to override this value? (2m should be enough time for Azure to become consistent)